stable/datadog/values.yaml

```diff
@@ -62,6 +62,12 @@ deployment:
   # Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
   tolerations: []
 
+  # You'll need to set dogstatsd node port.
```

Comment:
Could we do `If you're using a NodePort-type service and need a fixed port, set this parameter.` instead?

Reply:
Thank you to give nice comment!

https://github.com/kubernetes/charts/pull/6024/commits/ddc6240f7afbaf6989a9c63ba46f056b518f81b6